web/src/main/java/org/openmrs/web/filter/util/CustomResourceLoader.java

```diff
@@ -91,11 +91,12 @@ public static CustomResourceLoader getInstance(HttpServletRequest httpRequest) {
 	private Locale parseLocaleFrom(String filename, String basename) {
 		Locale result = null;
 		
+		String tmp_filename=filename;
```

Comment:
Your variable name does not match with our naming conventions https://wiki.openmrs.org/display/docs/Java+Conventions

You would also need to put some space before and after the assignment operator "="